CustomPlayerModels/src/shared/java/com/tom/cpl/gui/elements/Spinner.java

```diff
@@ -52,31 +54,58 @@ public void draw(MouseEvent event, float partialTicks) {
 		if(txtf.isFocused() && error != null) {
 			gui.drawRectangle(bounds.x, bounds.y, bounds.w, bounds.h, 0xffff0000);
 		}
+		// mouse click repeat
+		if (currentClick != null) {
+			mouseRepeatTimer -= partialTicks;
```

Comment:
We really shouldn't use the `partialTicks` for anything functional.
It'll break on many Minecraft versions (not even mojang knows what it should be https://bugs.mojang.com/browse/MC-273464)
On 1.21 NeoForge it's actually the partial ticks value, on Fabric and some older versions it's delta ticks from last frame,
but it changes depending on which minecraft version you are on.
And it makes the spinner buttons unusable, with high frame rates.

I'll have to go through and test every version and make sure it's the correct one (it makes the vanilla animations in the editor look jittery).

You can use:
`MinecraftClientAccess.get().getPlayerRenderManager().getAnimationEngine().getTicks()`
to get the current tick count from the game start.

Reply:
Done.